frontend/public/locales/fr/personal-information.json

```diff
@@ -132,29 +132,14 @@
     "edit": {
       "page-title": "(FR) Preferred language",
       "breadcrumbs": {
-        "personal-information": "(FR) Personal information"
+        "personal-information": "(FR) Personal information",
+        "preferred-language": "(FR) Preferred language",
+        "edit": "(FR) Edit"
       },
-      "preferred-language-legend": "(FR) What is your preferred language of communication?",
```

Comment:
Some i18n keys for the Edit page, including this one, have been removed but should remain. We should keep what's in `main` for the French text for the Edit page.

See https://github.com/DTS-STN/canadian-dental-care-plan/blob/d80126d4853e9c8313c4aa6229d554c1269afd2e/frontend/public/locales/fr/personal-information.json#L137

Reply:
good catch. fixed: 0b127dfed48b80758893e2181c4cd72cdf3b81a0